docs/spec/other/bech32.md

```diff
@@ -7,12 +7,12 @@ In the Cosmos network, keys and addresses may refer to a number of different rol
 
 ## HRP table     
 
-| HRP        | Definition |
+| HRP           | Definition |
 | ------------- |:-------------:|
-| `cosmosaccaddr`     | Cosmos Account Address     |
-| `cosmosaccpub`      | Cosmos Account Public Key  |
-| `cosmosvaladdr`     | Cosmos Consensus Address   |
-| `cosmosvalpub`      | Cosmos Consensus Public Key|
+| `cosa`        | Cosmos Account Address     |
```

Comment:
This does indeed shorten them, but I actually find this harder to reason about & read.

Reply:
`cosa` means thing in spanish

Reply:
I'm with @alexanderbez here. I don't mind how verbose they are because the descriptions are actually human readable.

Reply:
I agree; human-readable prefixes are more helpful for newer users, for whom distinguishability is most critical.

Reply:
++

Reply:
I still think we want to limit the cosmos account address prefix to as few chars as possible...
actually.. why not just "cosmos" for accounts, and cosmosX for others?
The main thing is for accounts, which I think can be given special consideration, e.g. I don't think it's necessary to append an "acc" or anything, it doesn't really help common users who won't care about the difference between validators and accounts... they'd mostly just assume that it's for an account.

cosmos
cosmospub
cosmosval
cosmosvalpub ?

Reply:
I agree on the new proposal @jaekwon

Reply:
I like that quite a bit. Much improved @jaekwon!

Reply:
++

Reply:
+++